cfme/configure/settings.py

```diff
@@ -20,6 +20,7 @@ class TimeProfileAddForm(View):
     days = BootstrapSwitch(name='all_days')
     hours = BootstrapSwitch(name='all_hours')
     save_button = Button('Save')
+    add = Button('Add')
```

Comment:
REQUIRED: please follow widget naming approach used in this view

Reply:
this is wrong way to add one more widget because title has been changed. use VersionPick for titles instead